sdk/src/main/scala/com/horizen/SidechainSettings.scala

```diff
@@ -25,7 +25,7 @@ case class GenesisDataSettings(scGenesisBlockHex: String,
                                mcNetwork: String,
                                withdrawalEpochLength: Int,
                                initialCumulativeCommTreeHash: String,
-                               isNonCeasing: Boolean = false
+                               isNonCeasing: Boolean
```

Comment:
Is there a check that this value has been set in the config?

Reply:
Yes, there are automatic checks for all settings that don't have default value.